user/user.go

```diff
@@ -0,0 +1,9 @@
+package user
+
+import "go.mongodb.org/mongo-driver/bson/primitive"
+
+type User struct {
+	ID       primitive.ObjectID `json:"id,omitempty" bson:"_id,omitempty"`
+	Nickname string             `json:"nickname"`
```

Comment:
Nickname uniq olsun. Uniq olması için chatGPT ye sorabilirsin

Bide Password alanı ekle

Reply:
bson bilgisi eksik. json yanına bson bilgiside eklenmeli